README.md

```diff
@@ -5,7 +5,7 @@
 * GLFW C library source is included and built automatically as part of the Go package. But you need to make sure you have dependencies of GLFW:
 	* On macOS, you need Xcode or Command Line Tools for Xcode (`xcode-select --install`) for required headers and libraries.
 	* On Ubuntu/Debian-like Linux distributions, you need `libgl1-mesa-dev` and `xorg-dev` packages.
-	* On CentOS/Fedora-like Linux distributions, you need `libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel libXi-devel libXxf86vm-devel` packages.
+	* On CentOS/Fedora-like Linux distributions, you need `libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel libXi-devel libXxf86vm-devel` packages. If you develop with wayland, you need `libxkbcommon-devel wayland-devel mesa-libGL-devel`.
```

Comment:
It seems a bit unclear if this means "you _only_ need" or "you _also_ need". Since "mesa-libGL-devel" is repeated, I'm guessing it's not the latter?

It might be more clear to reuse the phrasing from below:

"On CentOS/Fedora-like Linux distributions, you need [...common packages...]. To build for X, also need [...extra packages for X...]; and to build for Wayland, you also need [...extra packages for Wayland...]."

Reply:
It is also worth noting that `libdecor-devel` can be useful as an optional dependency for better titlebars.

Reply:
Sorry for the delay, work became hectic. Should be clearer !